node/src/env0-deploy-flow.js

```diff
@@ -14,19 +18,22 @@ const assertRequiredOptions = (options) => {
 }
 
 const runDeployment = async (command, options, environmentVariables) => {
+  const existingOptions = persistentOptions.read();
+
+  options = { ...existingOptions, ...options, }
   assertRequiredOptions(options);
 
   console.log(`running ${command} with the following arguments:`, options);
 
-  await DeployUtils.init(options);
-  switch(command) {
-    case 'deploy':
-      return await createAndDeploy(options, environmentVariables);
-    case 'destroy':
-      return await destroy(options);
-    default:
-      throw new Error(`Action ${command} is Invalid, Valid actions are Deploy and Destroy`);
```

Comment:
We don't validate the command is valid?

Reply:
because it already happens in `env0-deploy-cli.js` using the `assertCommandExists` method.